test/parallel/test-fs-stat-bigint.js

```diff
@@ -122,6 +122,28 @@ if (!common.isWindows) {
   fs.closeSync(fd);
 }
 
+{
+  assert.throws(
+    () => fs.statSync('does_not_exist'),
+    { code: 'ENOENT' });
+  assert.strictEqual(
+    fs.statSync('does_not_exist', { throwIfNoEntry: false }),
+    undefined);
+}
+
+{
+  assert.throws(
+    () => fs.lstatSync('does_not_exist'),
+    { code: 'ENOENT' });
+  assert.strictEqual(
+    fs.lstatSync('does_not_exist', { throwIfNoEntry: false }),
+    undefined);
+}
+
+{
+  assert.throws(() => fs.fstatSync(9999), { code: 'EBADF' });
```

Comment:
```suggestion
  assert.throws(() => fs.fstatSync(9999), { code: 'EBADF' });
  assert.throws(() => fs.fstatSync(9999, { throwIfNoEntry: false }), { code: 'EBADF' });
```

Reply:
I left that off because it's not a declared part of the API, but I see no harm in testing it.

Reply:
Oops, that caused a lint error.  Fix pending.